tester/runner.go

```diff
@@ -133,6 +133,21 @@ func (r *Runner) EnableTracing(yes bool) *Runner {
 // Run executes all tests contained in supplied modules.
 func (r *Runner) Run(ctx context.Context, modules map[string]*ast.Module) (ch chan *Result, err error) {
 
+	// rewrite duplicate test_* rule names
```

Comment:
I would consider factoring this out into a separate function but I'm fine with leaving it here for the time being.